languages/elixir/exercises/concept/booleans/lib/rules.ex

```diff
@@ -1,10 +1,17 @@
 defmodule Rules do
-  # TODO: implement the eat_ghost?/2 function
+  def eat_ghost?(power_pellet_active, touching_ghost) do
+    raise "eat_ghost?/2 not yet implemented"
```

Comment:
As we want this message to be as clear as possible to the student, maybe do:

```suggestion
    raise "Please implement the eat_ghost?/2 method"
```

If you like this, the same comment applies to the other stub methods.